dev-lang/lfe/lfe-1.2.1.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit multilib
+
+DESCRIPTION="Lisp-flavoured Erlang"
+HOMEPAGE="http://lfe.github.io/"
+SRC_URI="https://github.com/rvirding/lfe/archive/v${PV}.zip"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="dev-lang/erlang"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	export PATH="${S}/bin:$PATH"
+}
+
+src_compile() {
+	emake compile -j1
+}
+
+src_install() {
+	dobin bin/lfe
+	dobin bin/lfec
+	dobin bin/lfescript
+	dodir /usr/$(get_libdir)/erlang/lib/lfe/ebin/
+	dodir /usr/$(get_libdir)/erlang/lib/lfe/emacs/
+	cp -R "${S}/ebin" "${D}/usr/$(get_libdir)/erlang/lib/lfe/"
```

Comment:
1. `|| die`.
2. `${D}` contains a trailing slash already, so either `${D%/}/usr` or `${D}usr` (the former is more future-proof).
3. Unless i'm missing something, you should be able to collapse both `cp` calls into a single one.
4. I think `doins -r` would have the same effect here (unless I'm missing something), in which case it would be preferable over direct call to `cp`.